sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -539,16 +539,10 @@ case class BinaryPredicate(override val prettyName: String, left: Expression, ri
     Examples:
       > SELECT _FUNC_('Spark SQL', 'Spark');
        true
-      > SELECT _FUNC_(encode('Spark SQL', 'utf-8'), encode('Spark', 'utf-8'));
-       true
       > SELECT _FUNC_('Spark SQL', 'SPARK');
        false
-      > SELECT _FUNC_(encode('Spark SQL', 'utf-8'), encode('SPARK', 'utf-8'));
-       false
       > SELECT _FUNC_('Spark SQL', null);
        NULL
-      > SELECT _FUNC_(encode('Spark SQL', 'utf-8'), null);
-       NULL
       > SELECT _FUNC_(x'537061726b2053514c', x'537061726b');
```

Comment:
There is example for binary type already @beliefer

Reply:
I'm not familiar to the syntax x'537061726b2053514c'，I got it now.